### PR TITLE
Add attachment file

### PR DESCRIPTION
**Pull request by setmao**
Let we can add attachment file when build mail.
# Steps to Test This PR
The following steps are also performed and verified.

## Send The Rendered Emails with Attachment File
```
$python3 send_mail.py example/mail_config.json --attachment_file ~/Desktop/社群軌主題及場地.pdf
You are about to send the mails under "mails_to_sent". Do you want to continue? [y/N]: y      
Please enter your mail account: set840526@gmail.com
Please enter you mail password:
INFO:root:Email sent to example@gmail.com!
```

## Expected Result
The receivers could get the email with attachment file, and the file could open/download correctly.

### CODE REVIEW

**Comment by tai271828** [comment]
The feature is awesome and it will be useful to the sponsorship team definitely. There are two items still needing your help:

1. This pull request is very likely to raise change conflict after this pull request lands https://github.com/pycontw/mail_handler/pull/4 You may need to resolve the conflict later.
1. May you please elaborate how to test your pull request? You may refer to this pull request as an example https://github.com/pycontw/mail_handler/pull/4 (see "Steps to Test This PR")

**Comment by setmao** [comment]
Thx @tai271828 
I update more information for how to test this PR in first comment.

File: send_mail.py [hunk 1]
```diff
@@ -25,13 +27,25 @@ def load_mails(input_dir) -> Dict[str, str]:
 
 
 def build_mail(
-    receiver_addr: str, mail_content: str, config: Dict[str, str]
-) -> MIMEText:
-    mail = MIMEText(mail_content)
+    receiver_addr: str, mail_content: str, config: Dict[str, str], attachment_file=None
```
Comment (Lee-W) [inline send_mail.py:30]: I would suggest adding type annotation to `attachment_file` as well

File: send_mail.py [hunk 2]
```diff
@@ -25,13 +27,25 @@ def load_mails(input_dir) -> Dict[str, str]:
...
+) -> MIMEMultipart:
+    mail = MIMEMultipart()
+    mail.attach(MIMEText(mail_content))
     mail["Subject"] = config.get("Subject", "")
     mail["From"] = config.get("From", "")
     mail["To"] = receiver_addr
     mail["CC"] = config.get("CC", "")
+
+    if attachment_file is not None:
```
Comment (Lee-W) [inline send_mail.py:39]: `if attchement_file is not None:` seems to be unnecessary. Try `if attchement:`.

**Comment by setmao** [comment]
@Lee-W Thx for your suggest
I adding type annotation to attachment_file and use `if attchement:` in latest two commit.

**Comment by Lee-W** [comment]
The code looks good to me. Thanks for the contribution! The github action looks a bit weird now. I'll reword some of your commits and handle it on my side

**Pull request merged**